pkg/container/container.go

```diff
@@ -69,3 +63,51 @@ func CalculateID(c *Container) *ID {
 
 	return id
 }
+
+func (c *Container) Version() *pkg.Version {
+	return pkg.NewVersionFromV2(c.Container.GetVersion())
+}
+
+func (c *Container) SetVersion(v *pkg.Version) {
+	c.Container.SetVersion(v.ToV2())
+}
+
+func (c *Container) OwnerID() *owner.ID {
+	return owner.NewIDFromV2(c.Container.GetOwnerID())
+}
+
+func (c *Container) SetOwnerID(v *owner.ID) {
+	c.Container.SetOwnerID(v.ToV2())
+}
+
+func (c *Container) Nonce() []byte {
+	return c.Container.GetNonce() // return uuid?
```

Comment:
I think the question deserves a separate issue: what we do if `len(GetNonce) != uuid.Size`?